src/components/dagshub/data-engine/singleFileViewModal/SingleFileViewModal.tsx

```diff
@@ -38,7 +41,10 @@ export function SingleFileViewModal({
   onGetPreviousItemClickHandler,
   metadataOnChangeHandler,
   enableMetadataEditing,
-  enableMetadataDeletion
+  enableMetadataDeletion,
+  onSelectItem,
```

Comment:
Maybe it should be called onToggleItemSelection or something, cause it should be clear it is both for selecting and unselecting items